python/testing/__init__.py

```diff
@@ -127,6 +130,24 @@ def write_local_html_report(cls, report: str):
         if not QgisTestCase.is_ci_run():
             QDesktopServices.openUrl(QUrl.fromLocalFile(report_file))
 
+    @classmethod
```

Comment:
Shall this part not be in another PR ?

Reply:
Possibly, but let's avoid unnecessary load on the ci